docs/src/use/configure/migration-guide.md

```diff
@@ -674,6 +674,23 @@ You can see the TypeScript types for the flat config file format in the Definite
 
 You can view the type definitions in the [DefinitelyTyped repository on GitHub](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/eslint/index.d.ts).
 
+## Visual Studio Code Support
+
+By default, the Visual Studio Code ESLint plugin does not look for `eslint.config.js` files. To enable support for the new configuration files, edit your `.vscode/settings.json` file and add the following:
+
+```json
+{
+  /// enable flat config files (eslint.config.*js)
```

Comment:
Not technically incorrect, but: guessing this was meant to be the more common `//`?

```suggestion
  // enable flat config files (eslint.config.*js)
```